Dockerfile

```diff
@@ -17,7 +17,7 @@ WORKDIR /dscp-identity-service
 RUN npm -g install npm@8.x.x
 
 COPY . .
-RUN npm ci --production
+RUN --mount=type=secret,id=github GITHUB_PACKAGE_TOKEN=$(cat /run/secrets/github) npm ci --production
```

Comment:
I'm not sure this is needed

Reply:
looks like it is, `docker build` doesn't work locally for me without it either
https://github.com/digicatapult/dscp-identity-service/runs/6021152096?check_suite_focus=true